building/tracks/config-json.md

```diff
@@ -249,7 +249,33 @@ The key features are specified in the top-level `key_features` field which is de
 - `title`: a concise header for the key feature. Its length must be <= 25. Markdown is _not_ supported.
 - `content`: a description of the key feature. Its length must be <= 100. Markdown is _not_ supported.
 - `icon`: the icon to show for the feature. The following icons can be used:
-  - TODO: list icons
+  - `community`
+  - `concurrency`
+  - `cross-platform`
+  - `documentation`
+  - `dynamically-typed`
+  - `easy`
+  - `embeddable`
+  - `expressive`
+  - `extensible`
+  - `fast`
+  - `fun`
+  - `functional`
+  - `garbage`
+  - `general-purpose`
+  - `homoiconic`
+  - `interactive`
+  - `interop`
+  - `multi-paradigm`
+  - `portable`
+  - `powerful`
+  - `productive`
+  - `safe`
+  - `scientific`
+  - `shrink`
```

Comment:
What is `shrink`?

Reply:
I think that's the icon for a language producing small binanies. @taiyab?